src/sugar3/bundle/__init__.py

```diff
@@ -46,25 +47,31 @@
 
 Optional metadata keys are;
 
-* `summary` - a short summary of the activity that may be displayed in the List or Home Views,
+* `summary` - a short summary of the activity that may be displayed in
+  the List or Home Views,
 
 * `mime_types` - list of MIME types supported by the activity,
   separated by semicolons.  Your `read_file` method must be able to read
-  files of these MIME types.  Used to offer your activity when opening a
-  downloaded file or a journal object.
+  files of these MIME types.  Used to offer your activity in a list of activities
+  when opening a downloaded file from the web or a certain journal object.
```

Comment:
or from a USB drive, or by peer to peer file transfer, ... mentioning all the ways a journal object can be created seems a bit too much, and detracts from the core message.